### DISCUSSION TITLE
calm down iron golem fix

### DISCUSSION THREAD

**Discussion by danechek** [PurpurMC/Purpur#472] [__Dismissed__] [answered]
fix the calming of an aggressive iron golem with a rose so that the rose disappears from inventory when you give it to him

**Comment by BillyGalbreath** [comment, marked as answer]
Issues go in https://github.com/pl3xgaming/Purpur/issues ;)

I created a ticket for you https://github.com/pl3xgaming/Purpur/issues/476
